libraries/core-react/src/Chip/Chip.tsx

```diff
@@ -36,6 +35,8 @@ const StyledChips = styled.div.attrs(({ clickable, deletable }) => ({
   grid-auto-columns: max-content;
   align-items: center;
   z-index: 999;
+  tabIndex: ${({ clickable, deletable }) => (clickable || deletable ? 0 : null)},
```

Comment:
You can't rewrite it like this, as role and tabIndex is not CSS and it has to be in `attrs` function of styled components as it used to be. See e.g. MenuItem.tsx:
```
const ListItem = styled.li.attrs<StyleAttrsProps>(({ isFocused }) => ({
  role: 'menuitem',
  tabIndex: isFocused ? -1 : 0,
}))<StyleProps>`
  width: auto;
  position: relative;
  z-index: 2;

  ${typographyTemplate(typography)}
  ${spacingsTemplate(spacings)};
...
`
```

Reply:
Yes, I see that. Have fixed it so the properties now are assigned to attrs.